### PR TITLE
Update feature leads in README #1102

### PR DESCRIPTION
**Pull request by jamos-tay**
Feature leads updated.

Animation Lab - Ian
Color Lab - Ian
Zoom Lab - Shi Ya
Picture Slides Lab - Shi Ya
Narrations Lab - Ken
Captions Lab - Ken
Highlight Lab - Syed
Agenda Lab - Syed
Shapes Lab - Heng Le
Effects Lab - Heng Le
Position Lab - Jamos
Crop Lab - Jamos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,21 +49,21 @@ UI/UX | UI matters, UX matters | Ang Shi Ya | Ian | |
...
+AnimationLab (d) | Ian |  | Raunak
+CaptionsLab | Kenneth |  | Jerome
+ColorsLab | Ian |  | Karan
+CropLab | Jamos |  | Raunak
+EffectsLab | Heng Le |  | Fiona
+HighlightLab | Syed |  | Karan
+NarrationsLab | Ken |  | Jerome
+PasteLab | Syed |  |
+PictureSlideLab | Shi Ya |  | Xie Kai
+PositionsLab | Jamos |  | Oswell
```
Comment (damithc) [inline README.md]: Victor should be a past lead

File: README.md [hunk 2]
```diff
@@ -49,21 +49,21 @@ UI/UX | UI matters, UX matters | Ang Shi Ya | Ian | |
...
 ResizeLab | Min Oo | Yu Ting |  
-ShapesLab (d) |  |  | Huang Da
+ShapesLab (d) | Heng Le |  | Huang Da
 ShortcutsLab |  |  | Raunak
-SyncLab |  |  |
+SyncLab | Kenneth |  |
 ZoomLab |  |  | Raunak
```
Comment (damithc) [inline README.md]: Can @IanTeo take this? It's sort of similar to Animations Lab.

File: README.md [hunk 3]
```diff
@@ -49,21 +49,21 @@ UI/UX | UI matters, UX matters | Ang Shi Ya | Ian | |
...
+NarrationsLab | Ken |  | Jerome
+PasteLab | Syed |  |
+PictureSlideLab | Shi Ya |  | Xie Kai
+PositionsLab | Jamos |  | Oswell
 ResizeLab | Min Oo | Yu Ting |  
-ShapesLab (d) |  |  | Huang Da
+ShapesLab (d) | Heng Le |  | Huang Da
 ShortcutsLab |  |  | Raunak
-SyncLab |  |  |
+SyncLab | Kenneth |  |
```
Comment (damithc) [inline README.md]: Ian should be 2nd IC for this?

File: README.md [hunk 4]
```diff
@@ -49,21 +49,21 @@ UI/UX | UI matters, UX matters | Ang Shi Ya | Ian | |
...
-HighlightLab |  |  | Karan
-NarrationsLab |  |  | Jerome
-PasteLab |  |  |
-PictureSlideLab |  |  | Xie Kai
-PositionsLab | Victor |  | Oswell
+AgendaLab | Syed |  | Shunhao, Huang Da
+AnimationLab (d) | Ian |  | Raunak
+CaptionsLab | Kenneth |  | Jerome
+ColorsLab | Ian |  | Karan
+CropLab | Jamos |  | Raunak
```
Comment (damithc) [inline README.md]: Heng Le is 2nd IC of this?

File: README.md [hunk 5]
```diff
@@ -49,21 +49,21 @@ UI/UX | UI matters, UX matters | Ang Shi Ya | Ian | |
...
+EffectsLab | Heng Le |  | Fiona
+HighlightLab | Syed |  | Karan
+NarrationsLab | Ken |  | Jerome
```
Comment (damithc) [inline README.md]: Kenneth? (to be consistent)

File: README.md [hunk 6]
```diff
@@ -49,21 +49,21 @@ UI/UX | UI matters, UX matters | Ang Shi Ya | Ian | |
...
+PasteLab | Syed |  |
+PictureSlideLab | Shi Ya |  | Xie Kai
+PositionsLab | Jamos |  | Oswell
 ResizeLab | Min Oo | Yu Ting |  
-ShapesLab (d) |  |  | Huang Da
+ShapesLab (d) | Heng Le |  | Huang Da
 ShortcutsLab |  |  | Raunak
```
Comment (damithc) [inline README.md]: The two guys doing Sync Lab can do this one too (may be swap lead and co-lead)?

File: README.md (hunk 2 above)
Comment (IanTeo) [inline README.md]: If I am not mistaken, @AngShiYa is supposed to be the lead for ZoomLab.

**Comment by jamos-tay** [comment]
Okay, I've made the necessary changes

**Pull request merged**